ct/endurain.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+source <(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVED/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: johanngrobe
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/joaovitoriasilva/endurain
+
+APP="Endurain"
+var_tags="${var_tags:-sport;social-media}"
+var_cpu="${var_cpu:-2}"
+var_ram="${var_ram:-2048}"
+var_disk="${var_disk:-5}"
+var_os="${var_os:-debian}"
+var_version="${var_version:-13}"
+var_unprivileged="${var_unprivileged:-1}"
+
+header_info "$APP"
+variables
+color
+catch_errors
+
+function update_script() {
+  header_info
+  check_container_storage
+  check_container_resources
+
+  if [[ ! -d /opt/endurain ]]; then
+    msg_error "No ${APP} installation found!"
+    exit 1
+  fi
+
+  if check_for_gh_release "endurain" "joaovitoriasilva/endurain"; then
+
+    msg_info "Stopping Service"
+    systemctl stop endurain
+    msg_ok "Stopped Service"
+
+    msg_info "Creating Backup"
+    cp /opt/endurain/.env /opt/env.backup
```

Comment:
possibly this could be renamed to something else to not conflict with .env files by other apps/addons
```suggestion
    cp /opt/endurain/.env /opt/endurain.env
```